recipes/barrnap/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cp bin/* $PREFIX/bin
```

Comment:
Can this also be compiled from source? Should increase compatibility (since we are building on CentOS 5 for maximum compatibility with older libc).